### PR TITLE
Improve draggable slider thumbs

### PR DESCRIPTION
**Pull request by CodeTheory**
Bit of a large commit, but essentially this is some polish upon the trimmer to make it work as per the design.

### CODE REVIEW

File: app/src/renderer/js/editor.js [hunk 1]
```diff
@@ -92,7 +94,7 @@ document.addEventListener('DOMContentLoaded', () => {
     if (preview.duration) {
       const timeAtEvent = getTimestampAtEvent(event, preview.duration);
       previewTimeTip.style.left = `${event.pageX}px`;
-      previewTimeTip.textContent = `${moment().startOf('day').milliseconds(timeAtEvent * 1000).format('m:ss.SS')}`;
+      previewTimeTip.textContent = `${moment().startOf('day').milliseconds(timeAtEvent * 1000).format('m:ss.SS')} (${moment().startOf('day').milliseconds(currentPreviewDuration * 1000).format('m:ss.SS')})`;
```
Comment (albinekb) [inline app/src/renderer/js/editor.js]: I think we should have a common function to format these, I also think it's quicker to reuse the same moment objects rather than creating new ones, but this could be for another PR if we want to get this in quick 👍

Comment (CodeTheory) [reply app/src/renderer/js/editor.js]: Agreed, but for the beta, this should be fine 😄

**Comment by sindresorhus** [comment]
The initial trim handlers are a bit small before you hover the timeline. Not really clear that they are for trimming.

File: app/src/renderer/js/editor.js [hunk 2]
```diff
@@ -105,6 +107,13 @@ document.addEventListener('DOMContentLoaded', () => {
     }
   }
 
+  function getTrimmedVideoDuration() {
+    const inValue = getTrimmerValue(trimmerIn);
+    const outValue = getTrimmerValue(trimmerOut);
+    currentPreviewDuration = outValue - inValue;
```
Comment (albinekb) [inline app/src/renderer/js/editor.js]: this function mutates the external var `currentPreviewDuration`, better to make this one just 
```
return outValue - inValue
```
and mutate it where you call it further down
```
currentPreviewDuration = getTrimmedVideoDuration()
```

**Comment by skllcrn** [comment]
Should be clearer when we get the helper text implemented. I'll also keep tweaking the design based on beta feedback. 

editor

**Comment by sindresorhus** [comment]
The initial handles are much smaller here than in the above screenshot though.
screen shot 2017-10-31 at 23 59 55

File: app/src/renderer/css/editor.css [hunk 3]
```diff
@@ -146,19 +153,23 @@ video {
   transition: 200ms ease-out transform;
 }
 
-.timeline-container {
-  cursor: pointer;
+.timeline-container { cursor: pointer; }
```
[the remark this answers is not in the document]
Comment (CodeTheory) [reply app/src/renderer/css/editor.css]: I _only_ single line CSS that has one property because it's easier to scan through and helps with merge conflicts around it 😅

**Comment by skllcrn** [comment]
If we can merge this in for now, and make the design tweaks in the next beta release that'd be great.  Then I've got time to get it right.

File: app/src/renderer/js/editor.js (hunk 2 above)
Comment (sindresorhus) [reply app/src/renderer/js/editor.js]: @albinekb Can you do a follow-up PR with some improvements maybe?

Comment (CodeTheory) [reply app/src/renderer/js/editor.js]: Sorry, I missed this entirely.
